components/try.tsx

```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
         </div>
 
         {/* Content area */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
+        <div className="relative pb-2 pt-32 md:pb-4 md:pt-40">
           {/* Section header */}
           <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Try PearAI For Free
+            <h1 className="mb-4 text-4xl font-bold" data-aos="fade-up">
+              Interested in contributing to PearAI?
             </h1>
+            <p
+              className="text-secondary-400 mb-4 text-sm sm:text-lg"
+              data-aos="fade-up"
+              data-aos-delay="200"
+            >
+              Pear is built by a large community of developers. If you have
+              questions, or would like to discuss, you can join our{" "}
+              <Link
+                href="https://discord.gg/7QMraJUsQt"
+                className="hover:text-primary-800 text-primary-700"
+              >
+                Discord
+              </Link>{" "}
+              and talk to us directly! 🗣️
+            </p>
+            <p
+              className="text-secondary-400 mb-8 text-sm sm:text-lg"
+              data-aos="fade-up"
+              data-aos-delay="200"
+            >
+              Help the community out by giving{" "}
+              <Link
+                href="https://github.com/trypear/pearai-app"
+                className="hover:text-primary-800 text-primary-700"
```

Comment:
```suggestion
                className="hover:text-primary-800 font-medium text-primary-700"
```
Makes the clickable text easier to read.